data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/OperatorConfigurationTest.java

```diff
@@ -0,0 +1,57 @@
+package org.opensearch.dataprepper.expression;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensearch.dataprepper.expression.util.TestObject;
+
+import java.util.stream.Stream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+class OperatorConfigurationTest {
```

Comment:
Can we merge the test cases here into EqualOperatorTest and NotEqualOperatorTest?

Reply:
I scoped EqualOperatorTest and NotEqualOperatorTest to cover only the functionality of the corresponding class. GenericEqualOperator itself is not concerned with the result of comparing specific values until it is configured.

In my opinion testing the configured options here is better reflection of the separation of responsibility.